sale_readonly_security/views/sale_order_views.xml

```diff
@@ -5,6 +5,12 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="arch" type="xml">
+            <field name="partner_id" position="after">
```

Comment:
Seeing this change, this module is going to be very fragile if other fields in extra modules are added to the header and they are the same used in other places, so the strategy should switch to hide the header, but not remove it.

Reply:
Ok, changed to apply invisible when necessary.